src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -563,6 +597,23 @@ async Task<PurgeHistoryResult> IDurableOrchestrationClient.PurgeInstanceHistoryA
             return await this.DurabilityProvider.PurgeInstanceHistoryByInstanceId(instanceId);
         }
 
+        async Task<List<PurgeHistoryResult>> IDurableOrchestrationClient.PurgeInstanceHistoryAsync(IEnumerable<string> instanceId)
```

Comment:
I agree with you that it would be better to return a single `PurgeHistoryResult` object instead of returning a list. This is also consistent with the other `PurgeInstanceHistoryAsync` method that uses time and status filters to purge multiple instances.
```suggestion
        async Task<PurgeHistoryResult> IDurableOrchestrationClient.PurgeInstanceHistoryAsync(IEnumerable<string> instanceId)
```